make/golang.mk

```diff
@@ -147,6 +147,19 @@ $(call golang-vet,$(1))
 $(call golang-test-strict,$(1))
 endef
 
+# golang-build: builds a golang binary. ensures CGO build is done during CI. This is needed to make a binary that works with a Docker alpine image.
+# arg1: pkg path
+# arg2: executable name
+define golang-build
+@echo "BUILDING..."
+@if [ -z "$$CI" ]; then \
+	go build -o build/$(2) $(1); \
+else \
+	echo "-> Building CGO binary"; \
+	CGO_ENABLED=0 go build -installsuffix cgo -o build/$(2) $(1); \
```

Comment:
I have a preference for putting it in `bin/$(2)`, since
- latest template-worker, template-wag put things there
- consistency with bin/sfncli, bin/wag, bin/dep

Reply:
updated from build => bin

f98ac86